mono/mini/mini-arm64.c

```diff
@@ -5134,6 +5194,10 @@ emit_setup_lmf (MonoCompile *cfg, guint8 *code, gint32 lmf_offset, int cfa_offse
 	/* pc */
 	arm_adrx (code, ARMREG_LR, code);
 	code = emit_strx (code, ARMREG_LR, ARMREG_FP, lmf_offset + MONO_STRUCT_OFFSET (MonoLMF, pc));
+
+	const int num_save_lmf_instructions = 2;
+	mono_emit_unwind_op_prolog_nop(cfg, code, num_save_lmf_instructions); 
```

Comment:
Can we assert this constant matches what it corresponds to, i.e. that the lmf operation only uses two instructions? Does this correspond to the operation on line 5360?

Reply:
Changed to use a code size calculation instead of being hard coded in https://github.com/Unity-Technologies/mono/pull/1960/commits/7604e55851a7c96d7e06d1369e732d05a57b0733